procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,87 @@
+
+Recordings are usually used to ensure that the notes are correct and to optionally create automatic transcripts. Recordings can also *optionally* be made public.
+
+## Meeting roles
+
+The meeting has the following assigned roles:
+
+- **Facilitator**: the facilitator is in charge of pushing discussion along, calling on hands, formally asking the group whether there are any objections to a particular motion, and keeping the discussion on agenda.
+- **Scribe(s)**: the scribes are in charge of taking notes on what was discussed in the meeting including noting any action items or follow up questions.
+
+## Meeting Agenda
+
+The facilitator is in charge of ensuring that a proposal for the main agenda for the meeting is written into the notes document at least 2 days before the meeting. All Council members may comment on, revise, add to or otherwise collaborate on ensuring the agenda is in the right shape.
+
+The agenda should contain the following for each agenda item:
+
+- A short description of the contents of the item
+- The champion of this agenda item
+  - Usually this is the person who has suggested the agenda item and has the most context on what needs to be discussed.
+- An estimated time span for the item
+- A short description of the *goal(s)* of the item
+- An optional proposal (something that requires review and approval through the [decision making process][decisions])
+  - *Note*: proposals require another Council member to second the proposal before it can be brought forward.
+
+The agenda is laid out in temporal order which is determined largely by the urgency of each topic (most urgent comes first). However, this is not a hard rule as, for example, it might make sense to keep one large discussion (even if it is the most urgent) for last in the meeting so that it can simply fill the entire remainder of the meeting time.
+
+[decisions]: http://example.com (**TODO**: link when decision making process is documented)
+
+### Meeting structure
+
+In addition to the main agenda items, every meeting has the following structure:
+
+- **Check-in**
+  - The check-in is a chance for Council members to connect on a human level and share how their doing outside of the context of the meeting. Examples of things shared in the check-in include: "I'm not feeling well because of my allergies, and so I'm not in a good headspace", "My partner and I just adopted a cat so I'm doing great!", and "I'm doing fine - nothing to report".
+- **Assigning roles**
+  - The Council members decide by consensus who will fill the roles of facilitator and note taker.
+- **Turn on recording**
+  - Recording is started
+- **Consent to the agenda**
+  - This is a chance for last minute changes to the agenda to be surfaced and to ensure everyone is on the same page as far as what will be discussed.
```

Comment:
I think this should be an opportunity to have things struck from the agenda, for example if they are not in scope or there are legal issues which might mean a discussion is unwise. I'm not sure it should be an opportunity to add items for discussion as people will not have had time to consider them in advance of the meeting. If we do allow it, the addition of a late item should only be with the consent of the council (c.f. the addition of an item earlier in the process does not require the consent of the council).

Reply:
Ultimately everything (items added earlier in the process as well as last minute additions and deletions) is decided through consent (as the agenda must be consented to by everyone). I've added some notes that try to encourage keeping additions small and not pushing for additions unless consent can be agreed to quickly.